components/server/src/prometheus-metrics.ts

```diff
@@ -151,3 +151,25 @@ const instanceStartsFailedTotal = new prometheusClient.Counter({
 export function increaseFailedInstanceStartCounter(reason: "clusterSelectionFailed" | "startOnClusterFailed") {
     instanceStartsFailedTotal.inc({ reason });
 }
+
+const prebuildsStartedTotal = new prometheusClient.Counter({
+    name: "gitpod_prebuilds_started_total",
+    help: "Counter of total prebuilds started.",
+    labelNames: ["state"],
```

Comment:
Why do you need the state when the metric is explicitly for "started" prebuilds?